frontend/talentsearch/src/js/components/profile/ProfilePage/ProfilePage.tsx

```diff
@@ -55,7 +55,9 @@ export const ProfileForm: React.FC<ProfilePageProps> = ({
           backgroundRepeat: "no-repeat",
         }}
       >
-        <h1 data-h2-margin="b(top-bottom, l)">{`${firstName} ${lastName}`}</h1>
+        <h1 data-h2-margin="b(top-bottom, l)">
+          {firstName} {lastName}
```

Comment:
Didn't you want to put a placehoder here as Jerbo suggested?

Reply:
Yes I did and tested but didn't push😀